docs/DeveloperGuide.md

```diff
@@ -182,11 +182,14 @@ The `AddCommand` class is responsible for adding new patient's information in th
 #### Specifications
 
 * AddCommand, as defined by the `AddCommand` class, contain parameters which consists of: 
-  *  `patientHospitalId` integer, 
-  * `name`, `preferredName` String with only alphabets character,
-  *  `foodPreference`, `familyCondition`, `hobby` and `tag` which are alphanumeric.
+  *  `patientHospitalId` non-negative numeric characters which uniquely identifies the patient, 
```

Comment:
maybe it might be better to say String of non-negative numerics instead as people might be wondering why not use non-negative integer?